agent/commands/clients.go

```diff
@@ -200,9 +200,14 @@ func serverRegister(cfgSetup *config.Setup) (string, error) {
 		Context: context.Background(),
 	})
 	if err != nil {
-		return "", err
+		return "", "", err
 	}
-	return res.Payload.PMMAgent.AgentID, nil
+
+	if res.Payload.Token == "" {
+		return "", "", errors.New("server responded with empty token (please consider upgrade server version).")
```

Comment:
We shall make this error more descriptive. It's not clear what is expected of the user.

Reply:
let's not return an error, but just use the username and password that we already have.

Reply:
Maybe write some warning message, but let it work